sqlstore/src/main/resources/db/migration/V8003__oicr_sequencers.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO SequencerReference (referenceId, name, ipAddress, platformId, available, serialNUmber, dateCommissioned, dateDecommissioned, upgradedSequencerReferenceId) VALUES 
+(1, 'D00331', UNHEX('7F000001'), 16, 1, 'HSQ-D00331', '2014-01-03', NULL, NULL),
+(2, 'D00343', UNHEX('7F000001'), 16, 1, 'D00343', '2014-01-15', NULL, NULL),
+(3, 'D00353', UNHEX('7F000001'), 16, 1, NULL, '2013-12-17', NULL, NULL),
+(4, 'D00355', UNHEX('7F000001'), 16, 1, NULL, '2014-01-15', NULL, NULL),
+(5, 'SN1068', UNHEX('7F000001'), 16, 1, 'HSQ-7001068', '2012-02-03', NULL, NULL),
+(6, 'SN1080', UNHEX('7F000001'), 16, 1, 'HSQ-7001080', '2011-10-18', NULL, NULL),
+(7, 'SN7001179', UNHEX('7F000001'), 16, 1, 'HSQ-7001179', '2012-02-22', NULL, NULL),
+(8, 'SN7001205', UNHEX('7F000001'), 16, 1, 'HSQ-7001205', '2015-10-30', NULL, NULL),
+(9, 'SN203', UNHEX('7F000001'), 16, 1, 'HSQ-700203N', '2010-07-23', NULL, NULL),
+(10, 'h203', UNHEX('7F000001'), 16, 1, 'HSQ-700203N', '2010-07-23', '2010-10-22', 9),
+(11, 'SN231', UNHEX('7F000001'), 16, 1, NULL, '2010-07-23', '2010-10-04', NULL),
+(12, 'h239', UNHEX('7F000001'), 16, 1, NULL, '2010-07-23', NULL, NULL),
+(13, 'SN393', UNHEX('7F000001'), 16, 1, NULL, '2010-10-19', '2011-11-18', 12),
+(14, 'h393', UNHEX('7F000001'), 16, 1, NULL, '2010-10-19', '2012-02-01', NULL),
+(15, 'SN801', UNHEX('7F000001'), 16, 1, 'HSQ-700801', '2011-04-08', NULL, NULL),
```

Comment:
13 has a reference to 12, but based in the name, the reference should be to 14 instead. Can you confirm?

Additionally, it looks like in most cases, the SN-series references the h-series instrument with the same numeric suffix. In 9 and 10 this pattern is reversed. Can you confirm?